### DISCUSSION TITLE
ubluetooth missing from GENERIC_S3_SPIRAM-20220618-v1.19.1.bin

### DISCUSSION THREAD

**Discussion by AeroSynth** [micropython/micropython#10710] [ESP32]
I have two ESP32-S3 Wroom DevKitC boards.
One has 2K SPIRAM 8M Flash and the other only has the 8M Flash.
When I started testing GENERIC_S3_SPIRAM-20220618-v1.19.1.bin, the ubluetooth library seems to be missing.  It's not listed when you type help(modules).
However, the other image GENERIC_S3-20220618-v1.19.1.bin does include ubluetooth and it seems to function running the BLE examples.
I also noted that the SPIRAM image is smaller than the other one suggesting it may be missing some bits.

**Comment by mattytrentini** [comment]
Please don't raise duplicate discussions and [issues](https://github.com/micropython/micropython/issues/10711). 😜 

(Hopefully the detail I've added to the issue will sort you out.)

**Comment by AeroSynth** [comment]
OK.  Wasn't sure.
Brad
